### PR TITLE
Cherry picked exercises from bewatts/xelm

### PR DESCRIPTION
**Pull request by tgecho**
This is the last big exercise dump @parkerl, I promise!

After a bit of consultation with @bewatts, I cherry picked and tidied up a bunch of exercises from https://github.com/bewatts/xelm

- accumulate
- sublist
- sum-of-multiples
- strain
- space-age
- nucleotide-count
- phone-number
- grade-school

I tried to maintain the original authorship as much as possible (I think @clairethompson and a few others pairing!).

### CODE REVIEW

**Comment by parkerl** [comment]
Wow! Nice. I will try to get through them today. I'm growing fond of these commit bombs!

**Comment by parkerl** [comment]
Looks like `elm-format` thinks some of the examples need updating?

File: exercises/grade-school/GradeSchool.example [hunk 1]
```diff
@@ -0,0 +1,32 @@
+module GradeSchool (..) where
+
+import Dict exposing (..)
+
+type alias Grade = Int
```
Comment (parkerl) [inline exercises/grade-school/GradeSchool.example]: Example of where `elm-format` wants an extra line

**Comment by parkerl** [comment]
Not quite done reviewing but I think if these look good we should merge and then format? @tgecho

File: exercises/grade-school/GradeSchool.example (hunk 1 above)
Comment (tgecho) [inline exercises/grade-school/GradeSchool.example]: Yep. That's interesting... it turns out the elm-format sublime text plugin I'm using doesn't seem to do anything if the file doesn't actually have a `.elm` extension. I revised the grade-school revision commit (now 6b79102) with updated formatting. All of the other examples from this batch seem to be good.

**Comment by parkerl** [comment]
Booya! :tada:

**Pull request merged**